geaflow/geaflow-common/src/main/java/com/antgroup/geaflow/common/binary/BinaryString.java

```diff
@@ -372,6 +372,15 @@ public BinaryString substring(final int start, final int end) {
         }
     }
 
+    public BinaryString reverse() {
+        byte[] bytes = new byte[numBytes];
+
+        for (int i = 0 ; i < numBytes ; i++) {
+            bytes[i] = getByte(numBytes - i - 1);
```

Comment:
As BinaryString is a binary representation of a string, which might involve some compression techniques. Simply reversing it might not yield the expected results. Instead, it is necessary to first decode to the actual string, reverse it, and then reconstruct a new BinaryString.

Reply:
I tested it with a test case and found out it works, that's why I used reversing binary array to reverse the string. I'm wondering if decode BinaryString to an actual string and then convert it back, will there be extra overhead?

Reply:
Yes, I see that the simple examples added in the unit tests are feasible. However, the BinaryString interface is designed to enhance the String with a binary representation, which means its data structure may not always be a reversible byte array. Therefore, this overhead is necessary.

Reply:
Sure, will fix it.